<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1089</width>
    <height>690</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>B Nazanin</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::RightToLeft</enum>
  </property>
  <property name="locale">
   <locale language="Persian" country="Iran"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="grp_info">
     <property name="title">
      <string>اطلاعات مسئول</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="lbl_employee_id_title">
        <property name="text">
         <string>کد مسئول:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbl_employee_id">
        <property name="text">
         <string>1234</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="lbl_fname_title">
        <property name="text">
         <string>نام:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbl_fname">
        <property name="text">
         <string>نام</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="lbl_lname_title">
        <property name="text">
         <string>نام خانوادگی:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbl_lname">
        <property name="text">
         <string>نام خانوادگی</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="lbl_phone_title">
        <property name="text">
         <string>تلفن همراه:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbl_phone">
        <property name="text">
         <string>09131234567</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="lbl_section_title">
        <property name="text">
         <string>بخش:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbl_section">
        <property name="text">
         <string>نام بخش</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="lbl_is_manager_title">
        <property name="text">
         <string>وضعیت:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbl_is_manager">
        <property name="text">
         <string>مدیر</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hlay_operations">
     <item>
      <widget class="QGroupBox" name="grp_employee_operations">
       <property name="title">
        <string>عملیات مسئول</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="2">
         <widget class="QPushButton" name="btn_delete_member">
          <property name="text">
           <string>حذف عضو</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="btn_show_members">
          <property name="text">
           <string>مشاهده اعضا</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="btn_add_member">
          <property name="text">
           <string>افزودن عضو</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="btn_add_book">
          <property name="text">
           <string>ثبت کتاب</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="btn_delete_book">
          <property name="text">
           <string>حذف کتاب</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="btn_delete_old_books">
          <property name="text">
           <string>حذف کتاب‌های قدیمی</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="grp_manager_operations">
       <property name="title">
        <string>عملیات مدیر</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QPushButton" name="btn_show_employees">
          <property name="text">
           <string>مشاهده مسئولان</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="btn_stat_loans">
          <property name="text">
           <string>آمار امانت کتاب از بخش‌ها</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="btn_stat_members">
          <property name="text">
           <string>آمار فعالیت اعضا</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hlay_tables">
     <item>
      <widget class="QGroupBox" name="grp_loans">
       <property name="title">
        <string>تایید/رد درخواست‌های امانت</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QTableWidget" name="tbl_loan_requests">
          <column>
           <property name="text">
            <string>کد کتاب</string>
           </property>
           <property name="font">
            <font>
             <family>B Nazanin</family>
             <pointsize>12</pointsize>
            </font>
           </property>
          </column>
          <column>
           <property name="text">
            <string>عنوان</string>
           </property>
           <property name="font">
            <font>
             <family>B Nazanin</family>
             <pointsize>12</pointsize>
            </font>
           </property>
          </column>
          <column>
           <property name="text">
            <string>کد عضو</string>
           </property>
           <property name="font">
            <font>
             <family>B Nazanin</family>
             <pointsize>12</pointsize>
            </font>
           </property>
          </column>
          <column>
           <property name="text">
            <string>تاریخ درخواست</string>
           </property>
           <property name="font">
            <font>
             <family>B Nazanin</family>
             <pointsize>12</pointsize>
            </font>
           </property>
          </column>
          <column>
           <property name="text">
            <string>عملیات</string>
           </property>
           <property name="font">
            <font>
             <family>B Nazanin</family>
             <pointsize>12</pointsize>
            </font>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="grp_books">
       <property name="title">
        <string>لیست کتاب‌ها</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QTableWidget" name="tbl_books">
          <column>
           <property name="text">
            <string>کد کتاب</string>
           </property>
           <property name="font">
            <font>
             <family>B Nazanin</family>
             <pointsize>12</pointsize>
            </font>
           </property>
          </column>
          <column>
           <property name="text">
            <string>عنوان</string>
           </property>
           <property name="font">
            <font>
             <family>B Nazanin</family>
             <pointsize>12</pointsize>
            </font>
           </property>
          </column>
          <column>
           <property name="text">
            <string>نویسنده</string>
           </property>
           <property name="font">
            <font>
             <family>B Nazanin</family>
             <pointsize>12</pointsize>
            </font>
           </property>
          </column>
          <column>
           <property name="text">
            <string>انتشارات</string>
           </property>
           <property name="font">
            <font>
             <family>B Nazanin</family>
             <pointsize>12</pointsize>
            </font>
           </property>
          </column>
          <column>
           <property name="text">
            <string>بخش</string>
           </property>
           <property name="font">
            <font>
             <family>B Nazanin</family>
             <pointsize>12</pointsize>
            </font>
           </property>
          </column>
          <column>
           <property name="text">
            <string>وضعیت کتاب</string>
           </property>
           <property name="font">
            <font>
             <family>B Nazanin</family>
             <pointsize>12</pointsize>
            </font>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
